query/stdlib/influxdata/influxdb/rules.go

```diff
@@ -680,69 +680,19 @@ func (rule PushDownWindowAggregateRule) Pattern() plan.Pattern {
 }
 
 func canPushWindowedAggregate(ctx context.Context, fnNode plan.Node) bool {
```

Comment:
We're only pushing down mean as a part of this PR. Cases can be added back for each agg as it is ported.